CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# CMakeLists.txt - cmake build for leveldown
+# Copyright (c) 2020, Christopher Jeffrey (MIT License).
+# https://github.com/bcoin-org/bdb
+
+cmake_minimum_required(VERSION 3.4)
+project(leveldown LANGUAGES CXX)
```

Comment:
Might need `C CXX` here since we have a the `win_delay_load_hook.c` when building under Windows...

Reply:
win_delay_load_hook.c is valid C++, so it seems to work without it. I think it's just building it as C++.